src/measure_theory/borel_space.lean

```diff
@@ -704,6 +715,22 @@ hf.nnnorm.ennreal_coe
 
 end normed_group
 
+section normed_space
+
+variables [measurable_space α]
+variables {𝕜 : Type*} [normed_field 𝕜]
+variables {E : Type*} [normed_group E] [normed_space 𝕜 E] [measurable_space E] [borel_space E]
+variables {F : Type*} [normed_group F] [normed_space 𝕜 F] [measurable_space F] [borel_space F]
+
+lemma continuous_linear_map.measurable (L : E →L[𝕜] F) : measurable L :=
+L.continuous.measurable
+
+lemma measurable.clm_apply {φ : α → E} (φ_meas : measurable φ)
+  (L : E →L[𝕜] F) : measurable (λ (a : α), L (φ a)) :=
+L.measurable.comp φ_meas
```

Comment:
Those two lemmas, as well as `measurable_norm` and `integrable_norm` sound a bit silly. But integration theory is full of lemmas requiring measurability and integrability of many functions, and those proofs really get out of control quickly, so every shortcut is precious.